src/commands/org/open.ts

```diff
@@ -66,12 +67,13 @@ export class OrgOpenCommand extends OrgOpenCommandBase<OrgOpenOutput> {
   };
 
   public async run(): Promise<OrgOpenOutput> {
+    this.warn(sharedMessages.getMessage('BehaviorChangeWarning'));
     const { flags } = await this.parse(OrgOpenCommand);
     this.org = flags['target-org'];
     this.connection = this.org.getConnection(flags['api-version']);
 
     const [frontDoorUrl, retUrl] = await Promise.all([
-      buildFrontdoorUrl(this.org, this.connection),
+      buildFrontdoorUrl(this.org, this.connection, !(flags['url-only'] || this.jsonEnabled())),
```

Comment:
@EstebanRomero84 
from yesterday discussion:
* add support for boolean `SF_SINGLE_USE_ORG_OPEN_URL` env var, this should override either `--url-only`/`--json` so users can try the new behavior.

use the `Env` class to read boolean env vars:
https://github.com/search?q=repo%3Asalesforcecli%2Fplugin-org+%2Fenv%5C.getBoolean%2F&type=code